projects/ui/src/lib/components/po-dynamic/po-dynamic-view/po-dynamic-view.component.spec.ts

```diff
@@ -119,6 +152,115 @@ describe('PoDynamicViewComponent:', () => {
       expect(component['getValueFields']).toHaveBeenCalled();
     });
 
+    it(`setFieldOnLoad: should update field value if it is an existing field`, () => {
+      component.fields = [{ property: 'name', tag: true }];
+
+      const expectedValue = <any>{ property: 'name', tag: true, inverse: true };
+      const fakeFieldOnLoad = {
+        value: {},
+        fields: [
+          { property: 'name', inverse: true }
+        ]
+      };
+
+      spyOn(component, <any> 'getValueFields');
+
+      component['setFieldOnLoad'](fakeFieldOnLoad.fields[0]);
+
+      expect(component.fields[0]).toEqual(expectedValue);
+    });
+
+    it(`setFieldOnLoad: should update fields value if it is a new field`, () => {
+      component.value = { name: 'teste'};
+      component.fields = [{ property: 'name' }];
+
+      const fakeFieldOnLoad = {
+        value: { age: '20'},
+        fields: [
+          {
+            property: 'age',
+            tag: true,
+            inverse: true
+          }
+        ]
+      };
+
+      const expectedFields = [{ property: 'name' }, { property: 'age', tag: true, inverse: true }];
+
+      spyOn(component, <any> 'getValueFields');
+
+      component['setFieldOnLoad'](fakeFieldOnLoad.fields[0]);
+
+      expect(component.fields).toEqual(expectedFields);
+    });
+
+    it(`setValuesAndFieldsOnLoad: should call 'dynamicViewService.onLoad' and 'setValueOnLoad' if 'url' is truthy`, () => {
+      component.url = 'teste';
+
+      const dataOnLoad = {
+        value: { age: '20'},
+        fields: [
+          {
+            property: 'age',
+            tag: true,
+            inverse: true
+          }
+        ]
+      };
+
+      spyOn(component.dynamicViewService, 'onLoad').and.returnValue(getObservable(dataOnLoad));
```

Comment:
Podemos utilizar **of** no lugar do **getObservable**, com isso podemos remover o metodo no util-expect.spec.ts

```
spyOn(component.dynamicViewService, 'onLoad').and.returnValue(of(dataOnLoad));
```